packages/inventory-compliance/src/SystemRulesTable/SystemRulesTable.js

```diff
@@ -328,6 +328,12 @@ class SystemRulesTable extends React.Component {
             this.state.activeFilters,
             { hideLabel: true }
         );
+        const systemToRemediate = {
+            id: system.id,
+            profiles: system.testResultProfiles,
+            ruleObjectsFailed: [],
+            supported: system.supported
```

Comment:
This seems to break when creating a new policy in the rules step. `system` is `undefined`.

Reply:
@akofink Good catch! I wasn't considering the remediations disabled catch. It shouldn't break anymore.